src/worker/cannon-worker.ts

```diff
@@ -0,0 +1,465 @@
+import EventEmitter from 'events'
+
+import type {
+  AddBodiesMessage,
+  AddConstraintMessage,
+  AddContactMaterialMessage,
+  AddRaycastVehicleMessage,
+  AddRayMessage,
+  AddSpringMessage,
+  ApplyForceMessage,
+  ApplyImpulseMessage,
+  ApplyLocalForceMessage,
+  ApplyLocalImpulseMessage,
+  ApplyRaycastVehicleEngineForceMessage,
+  ApplyTorque,
+  AtomicMessage,
+  Broadphase,
+  CannonMessage,
+  CannonWebWorker,
+  DefaultContactMaterial,
+  DisableConstraintMessage,
+  DisableConstraintMotorMessage,
```

Comment:
I feel like we could have a single type that we import instead of each individually named message.

I'd like to remove the need for WithoutOp as well.

Maybe we can do something like:
`CannonMessageBody<'disableConstraintMotor'>`
or
`CannonMessageMap['disableConstraintMotor']`